modules/nodes-sigstore-configure-cluster-policy.adoc

```diff
@@ -69,6 +93,41 @@ spec:
 --
 +
 --
+.Example cluster image policy object for a BYOPKI policy and the `MatchRepository` match policy
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ClusterImagePolicy <1>
+metadata:
+  name: pki-policy
+spec:
+  scopes:
+  - example.io <2>
+  policy: <3>
+    rootOfTrust: <4>
+      policyType: PKI <5>
+      pki: <6>
+        caRootsData: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk....URS0tLS0t
+        caIntermediatesData: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1J....lDQVRFLS0tLS0=
+        pkiCertificateSubject: <7>
+          email: email@example.com
+          hostname: myhost.example.com
+    signedIdentity:
+      matchPolicy: MatchRepository <8>
+----
+<1> Creates a `ClusterImagePolicy` object.
+<2> Defines a list of repositories or images assigned to this policy. In a cluster image policy, make sure that the policy does not block the deployment of the {product-title} images in the `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev` repositories. Images in these repositories are required for cluster operation.
+<3> Specifies the parameters that define how the images are verified.
+<4> Defines a root of trust for the policy.
+<5> Specifies the policy types that define the root of trust, either a public key, a BYOPKI certificate, or a link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio certificate]. Here, a BYOPKI certificate.
```

Comment:
```suggestion
<5> Specifies the policy types that define the root of trust, either a public key, a BYOPKI certificate, or a link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio certificate]. Here, a BYOPKI certificate is used.
```
Feel free to take or leave this suggestion. The second sentence seems a bit incomplete as it is.